app/controllers/FeedbackFormsResponseController.scala

```diff
@@ -235,6 +235,7 @@ class FeedbackFormsResponseController @Inject()(messagesApi: MessagesApi,
 
       (question.questionType, question.mandatory) match {
         case ("MCQ", true)      => if (question.options.contains(response) && response.nonEmpty) {
+          Logger.info(s"The response was -------------------------> " + response)
```

Comment:
Can we get rid of this logger? It does not seem useful.

Reply:
Yes.